README.md

```diff
@@ -98,7 +98,7 @@ Because they are based on scripts in
 [PyGObject-Tutorial](https://github.com/sebp/PyGObject-Tutorial).
 [PyGObject-Tutorial is licensed under GFDL 1.3](https://github.com/sebp/PyGObject-Tutorial/blob/master/COPYING).
 
-See gtk3/examples/ directory for details.
+See [gtk3/sample/](/gtk3/sample) directory for details.
```

Comment:
```suggestion
See [gtk3/sample/](gtk3/sample/) directory for details.
```

Reply:
No, the point of having /gtk3/sample instead of gtk3/sample is to keep the path absolute in relation to the entire repository so that if you ever copy/move that line around, it continues to work.